lib/components/src/card/card-base.ts

```diff
@@ -17,29 +14,46 @@ export class CardBase extends LitElement {
    */
   @property()
   subTitle = '';
-
+  /**
+   * Displays the action bar
+   */
+  @property({type: Boolean}) actionBar = false;
   /**
    * Style the card as an outline variant
    */
   @property({type: Boolean}) outlined = false;
-   
   override render() {
     const classes = {
       'mdc-card': true,
       'mdc-card--outlined': this.outlined,
     };
     return html`
+      <style>
+        hr {
+          opacity: 0.12;
+          width: 100%;
+          margin: 0;
+        }
+        .mdc-typography--headline5 {
```

Comment:
Rather than defining a general typography variable like this and applying it in the markup, just apply the style directly to the element you want to modify. So find a way to target the mdc-card__actions item in your card, or some other semantic identifier.

Reply:
Should I avoid targeting mdc classes in general or is this just typography-specific. The only property I change about `mdc-typography--headline5` is the text color.

Reply:
Just general best practice to apply your styles directly to the thing you want to change. It means that you only have one thing to accomplish (edit the css) instead of two (edit the css and add the right class to the markup).

Basically the goal here is to have the styling be affected only via css, without having to do anything in the markup. Makes the whole thing less brittle. Those utility classes are useful in on-the-fly situations but if we're building a component for reuse it should be as self-contained as we can make it.

Reply:
So in general, if I use the material utility classes, I should avoid editing their CSS as well? In that case, I should just directly target the `div` that's using the `mdc-card__actions` class, right? 

My understanding is that we are using Material Design components but we are overwriting some of their styles to match Teradata's design system.